Makefile

```diff
@@ -0,0 +1,48 @@
+.PHONY: test_contract test_contract.rs test-project test all check-ink-wrapper \
+	check-test-project all-dockerized tooling build-builder build-node run-node
+
+build-builder:
+	docker build --tag ink-builder --file ci/Dockerfile.builder ci
+
+build-node:
+	docker build --tag aleph-onenode-chain --file ci/Dockerfile.aleph-node ci
+
+run-node: build-node
+	docker run --detach --rm --network host aleph-onenode-chain \
+		--name ink-wrapper-builder
+
+test_contract:
+	cd test_contract && cargo contract build --release
+	cd test_contract && cargo contract upload --suri //Alice --url ws://localhost:9944 || true
```

Comment:
I know you don't like it but I think we should be using `ink-docker` wherever possible. Now we're just inconsistent (i.e. everywhere else we do try to use it).

Reply:
I don't understand this comment. It all either happens inside `ink-docker` or it doesn't, where is the inconsistency?

Reply:
That we're building contracts using installed `cargo-contract` rather than the dockerized version. It is consistent _within_ this app but not compared to other workflows we have.